public/wp-content/themes/glific/src/scss/style.scss

```diff
@@ -55,6 +55,34 @@ body{
 	border: 3px solid $theme-pewter;
 }
 
+.get-started-content-box {
```

Comment:
If possible, can you make it generic? It will be good to use it in other places.

Reply:
Removed as not required anymore. Nice catch.